### DISCUSSION TITLE
Welcome to Virtual-Assistant Discussions!

### DISCUSSION THREAD

**Discussion by ritwik12** [ritwik12/Virtual-Assistant#111] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EMDBuenavista** [comment]
Hi! Also training for VA right now. It's nice that you made this discussion here. Kindly check my article on the need of virtual assistants nowadays and also my website:
https://emdbvirtualassistance.wordpress.com/2020/12/03/2020-the-year-of-digitization-digital-jobs-and-virtual-assistance/
https://emdbvirtualassistance.wordpress.com/

**Reply by ritwik12** [reply]
Nice article

**Comment by skandaar** [comment]
Hi, I can't execute virtual_assistant, here's what I got after issuing the "_sudo make install_" command :

**yassine@yassine-ThinkPad-X240:~/github/Virtual-Assistant$** sudo make install
_ln -sf /home/yassine/github/virtual-assistant/Virtual-Assistant/./build/virtual_assistant /usr/bin/virtual_assistant
ln -sf /home/yassine/github/virtual-assistant/Virtual-Assistant/events.dat /usr/bin/events.dat
ln -sf /home/yassine/github/virtual-assistant/Virtual-Assistant/config		/usr/bin/config
_Installation Complete: run using virtual_assistant__

**yassine@yassine-ThinkPad-X240:~/github/Virtual-Assistant$** virtual_assistant
_virtual_assistant: command not found_

**Reply by ritwik12** [reply]
which OS? what version? @skandaar. Does comiliing with gcc works?
